js/tests/unit/util.js

```diff
@@ -42,6 +42,50 @@ $(function () {
     assert.strictEqual(typeof Util.isElement({}) === 'undefined', true)
   })
 
+  QUnit.test('Util.getTransitionDurationFromElement should accept transition durations in milliseconds', function (assert) {
+    assert.expect(1)
+    var $div = $('<div style="transition: all 300ms ease-out;"></div>').appendTo($('#qunit-fixture'))
```

Comment:
This test do not work, I don't know how but your `300ms` is converted into seconds in your JS, see our coverage : 

![image](https://user-images.githubusercontent.com/1689750/37308880-8bfe254c-263f-11e8-935f-709e472d837f.png)

Reply:
What did you do to detect this? `npm test` doesn't return any errors and the visual tests look fine?

Reply:
It's the coverage, see https://github.com/twbs/bootstrap/pull/25783

Reply:
My 2 cents are, that the condition it should test should be: `transitionDuration.indexOf('ms') != -1`

Reply:
Indeed, `indexOf` returns -1 if not found. So `!== -1` would be better.

Reply:
Changed it to `transitionDuration.indexOf('ms') !== -1`, problem with coverage still exists.

Reply:
As I said on my first message, this line : 
```js
let transitionDuration = $(element).css('transition-duration')
```

auto convert `transition-duration` into seconds.

IMO jQuery use `getComputedStyle` which do that

Reply:
jQuery takes care of this indeed (Also tested this in Chrome, FF, IE).

IMO it's safe to remove this check as long as we depend on jQuery.

Reply:
I don't think it's jQuery which do that, IMO it's more `getComputedStyle` but that's OK to me 👍